braph2genesis/graphs/_MultilayerWU.gen.m

```diff
@@ -90,7 +90,7 @@ In a multilayer weighted undirected (WU) graph, layers could have different numb
 value = Graph.NONNEGATIVE * ones(layernumber);
 
 %%% ¡prop!
-A (result, cell) is the cell containing the multilayer weighted adjacency matrices of the multilayer weighted undirected graph.
+A (result, cell) is the cell containing the multilayer weighted supra-adjacency matrices of the multilayer weighted undirected graph.
```

Comment:
A (result, cell) is the cell containing the multilayer weighted supra-adjacency matrix.